### PR TITLE
scarlin90-belfast

### PR DESCRIPTION
**Pull request by scarlin90**
This pull request adds a file about Belfast, Bel and where to grab some delicious food and beer! 🍺

### CODE REVIEW

File: scarlin90-belfast.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Belfast, Bel
+
+## Places to Eat
+- Boojum :buritos and :nachos
+- Laverys for :beers
```
Comment (githubteacher) [inline scarlin90-belfast.md:5]: @scarlin90 I think you need to add an ending `:`  to "beers"

File: scarlin90-belfast.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Belfast, Bel
+
+## Places to Eat
+- Boojum :buritos and :nachos
```
Comment (githubteacher) [inline scarlin90-belfast.md:4]: Don't forget to end emoji with an ending or trailing `:`

File: scarlin90-belfast.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+- Laverys for :beers
+- Made In Belfast for :dinner
+
```
Comment (ghost) [inline scarlin90-belfast.md:7]: Don't forget your emoji!

File: scarlin90-belfast.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Belfast, Bel
+
+## Places to Eat
+- Boojum :buritos and :nachos
+- Laverys for :beers
+- Made In Belfast for :dinner
```
Comment (githubteacher) [inline scarlin90-belfast.md:6]: One more example of the ending `:`

File: scarlin90-belfast.md (hunk 2 above)
Comment (ghost) [inline scarlin90-belfast.md:4]: YUUUUMMMMMMM

File: scarlin90-belfast.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+
+## Places to Visit
+
```
Comment (githubteacher) [inline scarlin90-belfast.md:10]: I've never been, I really like the craft beer scene, are there any breweries that have good tours?

File: scarlin90-belfast.md (hunk 4 above)
Comment (ghost) [inline scarlin90-belfast.md:6]: Here is another one just for kicks.

**Review by githubteacher — CHANGES_REQUESTED** [review]
Can you just address these comments before it is ready to be merged (:ship:)? 👍

**Review by ghost — COMMENTED** [review]
Please look at my comments (:ship:)?

**Review by JohnTresadern — CHANGES_REQUESTED** [review]
get that fixed pronto

**Review by JohnTresadern — DISMISSED** [review]
@scarlin90 please fix

**Comment by scarlin90** [comment]
I have fixed the beer imoji with a trailing ';'

File: scarlin90-belfast.md (hunk 2 above)
Comment (JohnTresadern) [inline scarlin90-belfast.md:4]: spelt burrito incorrectly

**Comment by scarlin90** [comment]
Merging my pull request

**Pull request merged**